reference-implementation/to-upstream-wpts/transform-stream/constructor.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+test(() => {
+  const ts = new TransformStream({ transform() { } });
+}, 'TransformStream can be constructed with a transform function');
+
+test(() => {
+  const error = new TypeError('something bad');
+  assert_throws(error, () => new TransformStream(), 'TransformStream cannot be constructed with no arguments');
```

Comment:
Nit: Use `assert_throws(new TypeError(), ...)` here instead of constructing an error with an arbitrary string.